sources/packages/web/src/services/AuthService.ts

```diff
@@ -67,6 +74,16 @@ export class AuthService {
    */
   priorityRedirect?: LocationAsRelativeRaw = undefined;
 
+  /**
+   * Function that makes a request to the API to log that the user has closed the browser/tab.
+   * @param event event received by the browser.
+   */
+  logUserClosedBrowser = async function (event: Event) {
+    event.preventDefault();
+    sessionStorage.setItem(USER_CLOSED_BROWSER, "true");
+    await ApiClient.AuditApi.audit(AuditEvent.BrowserClosed);
```

Comment:
I see that this gets called before clicking on the `Leave` of the dialog confirmation from the browser. Just trying to say that there is a possibility that the user cancels the page reload from the dialog and still this will be logged as user closed the browser. Under best effort, we may still be willing to keep this behavior. 

<img width="1917" alt="image" src="https://github.com/user-attachments/assets/fd502df7-cd4c-4f0d-818d-a44451bb003e">

Reply:
I removed that confirmation page that was not required. Thanks.